common/utils/src/main/resources/error/error-classes.json

```diff
@@ -97,6 +97,12 @@
       "The method <methodName> can not be called on streaming Dataset/DataFrame."
     ]
   },
+  "CANNOT_ALTER_PARTITION_COLUMN" : {
+    "message" : [
+      "ALTER TABLE (ALTER|CHANGE) COLUMN is not supported for partition columns, but found the partition column <columnName> in the table <tableName>"
+    ],
```

Comment:
Would it be good to clarify that it is not supported with the catalog of the specified table? e.g. sometimes it is supported with the right catalog.

Reply:
This error can come only from the command called for tables in V1 session catalog. The info about the catalog might confuse users, I guess.